File: NET/JNetCall.Sharp.Tests/NativeCallTest.cs
// ReSharper disable UnusedMember.Global
using JNetCall.Sharp.Client;
using static JNetCall.Sharp.Client.ServiceEnv;

namespace JNetCall.Sharp.Tests
{
    public sealed class NativeCallTest : CallTest
    {
        internal readonly string Path
            = BuildPath(@"..\..\JVM\alien2-java\target\alien2-java.jar");

        protected override T Create<T>()
        {
            return InProcClient.Create<T>(Path);
        }
    }
}

File: JVM/jnetcall-java-tests/src/test/java/jnetcall/java/tests/NativeCallTest.java
package jnetcall.java.tests;

import static jnetcall.java.client.tools.ServiceEnv.buildPath;

import jnetcall.java.client.InProcClient;

public final class NativeCallTest extends CallTest {

    final String Path =
        buildPath("..\\..\\..\\NET\\Alien2.Sharp\\bin\\Debug\\net6.0\\Alien2.Sharp.dll");

    @Override
    protected <T extends AutoCloseable> T create(Class<T> clazz) {
        return InProcClient.create(clazz, Path);
    }
}
